src/meltano/core/bundle/settings.yml

```diff
@@ -231,5 +231,9 @@ settings:
 - name: ff.strict_env_var_mode
   kind: boolean
   value: False
+# Whether invoke, and run operations if a plugin is not custom or it is not locked.
+- name: ff.plugin_locks_required
+  kind: boolean
+  value: false
```

Comment:
In addition to adding the feature flag here, it should be added to the `meltano.yml` schema at `src/meltano/schemas/meltano.schema.json`.

Reply:
Done! 772de9d